src/main/java/org/parabot/core/forum/AccountManager.java

```diff
@@ -103,6 +104,7 @@ public final boolean login(final String user, final String pass, boolean request
                 String api = (String) ((JSONObject) result.get("data")).get("api");
                 account = new Account(user, pass, api);
                 Context.setUsername(user);
+                new PBLocalPreferences("username.json").adjustSetting("username", user);
```

Comment:
I'd say rename it to `login.json`

Reply:
And maybe store the preference as a public variable instead of recreating it multiple times